### ISSUE TITLE
Image not showing up ReadMe once committed to GitHub

### ISSUE THREAD

**Issue by fbrown222** [Media-Ed-Online/intro-web-dev#11]
Once again, I am having troubles for my ReadME with my image not showing up on GitHub. My Markdown preview shows the image and my image is embedded in my Project_1 directory, so I can't figure out why it isn't showing. Anyone have a clue or having the same trouble?

**Comment by JonSwallow** [comment]
@fbrown222 Okay I went back and looked at your code and compared it to mine. This is the only thing I can figure.
*My Directory Windows 10*

![Screenshot of my Directory as directed](./images/screenshot-1.png)   <----- What worked for me.



And to finish, here is my screenshot: ![Screenshot Of my Directory](./images/Screen_Shot_1.png)       (here might be the problem)


And to finish, here is my screenshot:                                 <------------- (try this )

![Screenshot Of my Directory](./images/Screen_Shot_1.png)

**Comment by fbrown222** [comment]
@JonSwallow This unfortunately did not work either 👎 I tried separating them but no results still. I looked at your code as well, and did the same you did, but still failing. I wonder what this could be.

**Comment by JonSwallow** [comment]
@fbrown222 Wow I can not figure it out either. The only difference I see is that you have 2 repositories, however your address is good so I don't see how that could affect it. All the code looks the same. I even pasted yours into my atom to check it and it should work. I am at a loss sorry. Only thing I could suggest is maybe having only one repository. You are linked to the main one for the course in your repositories. You don't really need to have that there because you can get to it from here on the code button on top. I would just keep the one called WebDevHW.

**Comment by fbrown222** [comment]
@JonSwallow Still no luck even after the deletion. Thank you for testing it out with me and taking the time to help. I really do appreciate it!!! 👍 

@JustineEvansUM Do you possibly know what is happening here?

**Comment by JustineEvansUM** [comment]
@fbrown222 Are you ready for this?

I am embarrassed to admit that it look me nearly an hour and a half to notice you had a file naming error. That's all. I forked your directory and combed through your files and everything.

Your raw README has this code, `![Screenshot Of my Directory](./images/Screen_Shot_1.png)`, but your image directory is with a capital 'I'.

Try `![Screenshot Of my Directory](./Images/Screen_Shot_1.png)` and let me know how it goes!

**Comment by JonSwallow** [comment]
I would not feel so bad I looked it over at least five times and still missed it. I am a man I can change if I have to I guess.

**Comment by fbrown222** [comment]
@JustineEvansUM @JonSwallow AAAHHH!!!! I can't believe that is all that was!!! I am never capitalizing again!!! 
Thank you so much for your help!!!!!! Both of you!!! 👍 :D

**Comment by JohnYingling** [comment]
Yeah...quickly realizing we should follow all instructions to a T, including lowercase and "-" inputs.

Blergh.

**Comment by JustineEvansUM** [comment]
@fbrown222 Always a pleasure. This is how we learn!

(If it helps, I do this work day-in and day-out and I still make this error several times a week.)
